src/date-range-picker/calendar/grids/grid.tsx

```diff
@@ -2,14 +2,26 @@
 // SPDX-License-Identifier: Apache-2.0
 import React, { useMemo } from 'react';
 import styles from '../../styles.css.js';
-import GridDay from './day';
```

Comment:
Removed the grid day file for the layout to be only defined in one file avoiding the extra interfaces. That is consistent with Calendar grid implementation.